src/tests/java/htsjdk/samtools/metrics/MetricsFileTest.java

```diff
@@ -156,6 +158,45 @@ public void testWriteMetricsFile() throws IOException, ClassNotFoundException {
         histo.increment(1000, 10981982);
         file.setHistogram(histo);
 
+        //Get same histogram from file using two different methods
```

Comment:
space after `//` for each.  Makes it a lot easier to read, but I will leave it up to you.